src/internal/digitalAsset.ts

```diff
@@ -461,10 +540,15 @@ export async function unfreezeDigitalAssetTransaferTransaction(args: {
       functionArguments: [digitalAssetAddress],
     },
     options,
+    abi: unfreezeDigitalAssetAbi,
   });
-  return transaction;
 }
 
+const setDigitalAssetDescriptionAbi: EntryFunctionABI = {
```

Comment:
might be nice to define a DigitalAsset json ABI locally and access each relevant field, just to make it more readable and maintainable. not blocking tho

Reply:
You mean for the whole module, with each of the functions individually?

Reply:
I will come back to this when I go and fix it for ANS in a separate PR.